src/opnsense/mvc/app/models/OPNsense/Routes/Route/Migrations/M0_0_1.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace OPNsense\Routes\Route\Migrations;
+use OPNsense\Base\BaseModelMigration;
+class M0_0_1 extends BaseModelMigration
+{
+    public function run($model)
+    {
+       $model.save();
```

Comment:
this looks a bit off, If I'm not mistaken it should add uuid without an empty stub as well (as long as the version number is higher)